example_analysis/3.configure_phenotype_process_params.ipynb

```diff
@@ -188,29 +188,29 @@
     "### Segmentation\n",
     "\n",
     "- `DAPI_INDEX`: The index of the DAPI channel in your image stack (0-indexed). Typically 0 for the DAPI channel from sequencing.\n",
-    "- `CYTO_CHANNEL`: The index of the cytoplasmic channel in your image stack (0-indexed). Typically the channel that best outlines cell boundaries, usually base C.\n",
```

Comment:
Good to rename this because it is an index. But would it make sense to have it as a channel, and then call that from the list of provided channels in the notebook/snakemake? That may make it easier for users to understand. Up to you though.

Reply:
Good point! Done in https://github.com/cheeseman-lab/brieflow/pull/19/commits/252b80b92623b56e5f1c244306d956e31b5dd2e3